package-lock.json

```diff
@@ -2842,8 +2842,37 @@
 				"@babel/runtime": "^7.3.1",
 				"@wordpress/escape-html": "file:packages/escape-html",
 				"lodash": "^4.17.11",
-				"react": "^16.6.3",
-				"react-dom": "^16.6.3"
+				"react": "^16.8.4",
+				"react-dom": "^16.8.4"
+			},
+			"dependencies": {
```

Comment:
This part should be manually removed. The same versions of packages are now in the root of the lock file.

Reply:
Why do we need to do these manual tweaks? It seems to work as is?

Reply:
Oh I think it's because they're considered "dev" dependencies at the root level, this means `npm` marks them as ""dev": true," at the root level but also add them here as normal dependencies. I think we should keep them as is.